### ISSUE TITLE
Delete TXT records only added to the getcert-wilddns-with-zoneedit.sh script

### ISSUE THREAD

**Issue by amhuber** [jeansergegagnon/zoneedit_letsencrypt#17]
I've got certbot scheduled to run automatically and it's calling `certbot-dns-updater-with-zoneedit.sh`, which then calls `zoneedit.sh`.  The most recent run failed due to ZoneEdit adding two factor auth, so I pulled your latest commits to switch to a DYN token, which did work, but it is adding more TXT records every time I run it and not cleaning up the old ones.

I see you added the code at https://github.com/jeansergegagnon/zoneedit_letsencrypt/blob/master/getcert-wilddns-with-zoneedit.sh#L207-L240 to remove the old records, but that's only running if you use the `getcert-wilddns-with-zoneedit.sh` script, which I'm not (specifically because I include both *.domain.com and domain.com as SANs in my cert).

It looks like the `getcert-wilddns-with-zoneedit.sh` triggers a certbot run, which then calls `certbot-dns-updater-with-zoneedit.sh` -> `zoneedit.sh`, so it looks like if you just move the delete code into one of those scripts it would get called in all scenarios instead of just when using `getcert-wilddns-with-zoneedit.sh`?

**Comment by amhuber** [comment]
I just looked at the `getcert` script and realized it would work for me since it's adding both *.domain.com and domain.com, so I don't recall why I didn't use that script originally.  Hopefully you agree that moving the delete code makes sense to handle both scenarios correctly.  If you agree I'm also happy to submit a PR to resolve the issue.

**Comment by jeansergegagnon** [comment]
Oh hey Aaron,

Yeah, that's a good idea.

The problem is that those 2 scripts get called for each TXT record add
calls and it might delete the other just added TXT record.

I could make it delete 1 day old (or more) TXT records I guess but it
seemed more proper to delete in the domain update script.

Would you be able to use the other script?

**Comment by amhuber** [comment]
Yeah, that's reasonable.  I'll figure out how to turn off the scheduled run of certbot and test using the wrapper script on the next renew and make sure it's working for me.  I'll leave this open until then and close it if I don't have any issues.

**Comment by amhuber** [comment]
Sorry, forgot to close this.
